packaging/rhel/8/libreswan.spec

```diff
@@ -27,7 +27,7 @@
     USE_LIBCURL=true \\\
     USE_LINUX_AUDIT=true \\\
     USE_NM=true \\\
-    USE_NSS_KDF=true \\\
+    USE_NSS_KDF=false \\\
```

Comment:
these seem accidental, like the kvm-targets.mk and makefile changes. But why did you need to set this to false? As rhel8/centos8 should already contain new enough nss libraries to leave this set to true

Reply:
I will try reverting to see if it works. I can't recall, but this might be an artifact from when I was working on RHEL 8.2.